client/me/security-checkup/account-email.jsx

```diff
@@ -35,6 +45,19 @@ class SecurityCheckupAccountEmail extends Component {
 					},
 				}
 			);
+		} else if ( emailChangePending ) {
+			icon = getWarningIcon();
+			description = translate(
+				'You are in the process of changing your account email address to {{strong}}%(newEmailAddress)s{{/strong}}, but still need to confirm the change.',
```

Comment:
Should we add the 'you' word?

`but you still need to confirm the change.`

Reply:
Your suggestion is definitely clearer - I'll add the "you" to help everyone. 😁